build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/transport/TransportVersionResourcesService.java

```diff
@@ -107,6 +112,19 @@ TransportVersionDefinition getReferableDefinitionFromMain(String name) {
         return getMainFile(resourcePath, TransportVersionDefinition::fromString);
     }
 
+    List<String> getChangedReferableDefinitionNames() {
```

Comment:
This could use a comment. I'm not quite sure what "changed referrable definition" means.